CHANGELOG.md

```diff
@@ -463,6 +463,35 @@ Note that values that have been passed through the `$interpolate` service within
 `URL` or `MEDIA_URL` will have already been sanitized, so you would not need to sanitize
 these values again.
 
+  - due to **[1e9ead](https://github.com/angular/angular.js/commit/1e9eadcd72dbbd5c67dae8328a63e535cfa91ff9)**: handle URL sanitization through the `$sce` service
+
+binding `trustAs()` and the short versions `trustAsResourceUrl()` et. al. to
```

Comment:
I would somehow separate the "`trustAsResourceUrl()` et. al." part from the rest of the sentence. E.g. wrap it in `(...)`, `, ... ,`, `- ... -`.